riak/codecs/ttb.py

```diff
@@ -133,7 +130,7 @@ def encode_timeseries_query(self, table, query, interpolations=None):
         if '{table}' in q:
             q = q.format(table=table.name)
         tsi = tsinterpolation_a, q, []
-        req = tsqueryreq_a, tsi, False, []
+        req = tsqueryreq_a, tsi, False, udef_a
```

Comment:
@alexmoore - looks like Java client is correct:

https://github.com/basho/riak-java-client/blob/perf/ttb_encoding/v4/src/main/java/com/basho/riak/client/core/codec/TermToBinaryCodec.java#L77-L78